src/main/java/com/rarchives/ripme/ripper/rippers/InstagramRipper.java

```diff
@@ -212,23 +212,25 @@ private String getOriginalUrl(String imageURL) {
             JSONArray datas = new JSONArray();
             try {
                 JSONArray profilePage = json.getJSONObject("entry_data").getJSONArray("ProfilePage");
-                datas = profilePage.getJSONObject(0).getJSONObject("user").getJSONObject("media").getJSONArray("nodes");
+                datas = profilePage.getJSONObject(0).getJSONObject("graphql").getJSONObject("user")
+                        .getJSONObject("edge_owner_to_timeline_media").getJSONArray("edges");
             } catch (JSONException e) {
                 // Handle hashtag pages
                 datas = json.getJSONObject("entry_data").getJSONArray("TagPage").getJSONObject(0)
                         .getJSONObject("tag").getJSONObject("media").getJSONArray("nodes");
             }
             for (int i = 0; i < datas.length(); i++) {
                 JSONObject data = (JSONObject) datas.get(i);
-                Long epoch = data.getLong("date");
+                data = data.getJSONObject("node");
+                Long epoch = data.getLong("taken_at_timestamp");
                 Instant instant = Instant.ofEpochSecond(epoch);
                 String image_date = DateTimeFormatter.ofPattern("yyyy_MM_dd_hh:mm_").format(ZonedDateTime.ofInstant(instant, ZoneOffset.UTC));
                 if (data.getString("__typename").equals("GraphSidecar")) {
                     try {
-                        Document slideShowDoc = Http.url(new URL ("https://www.instagram.com/p/" + data.getString("code"))).get();
+                        Document slideShowDoc = Http.url(new URL ("https://www.instagram.com/p/" + data.getString("shortcode"))).get();
```

Comment:
Consistency error: `new URL (` -> `new URL(`

Reply:
Fixed in my last push